beacon_node/execution_layer/src/lib.rs

```diff
@@ -606,132 +652,193 @@ impl<T: EthSpec> ExecutionLayer<T> {
                         "Requesting blinded header from connected builder";
                         "slot" => ?slot,
                         "pubkey" => ?pubkey,
-                        "parent_hash" => ?parent_hash,
+                        "parent_hash" => %parent_hash,
```

Comment:
The `%` form is the short-form: https://github.com/sigp/lighthouse/pull/3725#discussion_r1033178270

Reply:
Ah of course 🤦 Fixed in e8bd2be26